templates/cluster/vsphere-standalone-cp/templates/vspheremachinetemplate-worker.yaml

```diff
@@ -17,8 +17,9 @@ spec:
             networkName: {{ .Values.worker.network }}
           {{- if .Values.ipamEnabled  }}
             addressesFromPools:
-             - {{- toYaml .Values.ipPool.data | nindent 14 }}
+             - {{- toYaml .Values.ipPool.config | nindent 14 }}
```

Comment:
Oh, I missed that: I reckon this should be removed, shouldn't it?

Reply:
This is needed but it's provided in code during the ipam claim processing.